src/styles/layout.sss

```diff
@@ -8,21 +8,18 @@
     bottom: 0
 
 .layout__header
-    background: #222
+    background: #202020
     flex: 1 0
-    max-height: 2.5em
-    z-index: 500 //To overshadow everything
-    box-shadow: 3px 3px 3px 4px rgba(0, 0, 0, 0.4)
-    //Text properties
-    font-size: 1.25em
+    max-height: 3.1em
+    z-index: 500 // To overshadow everything
     color: white
-    //Elements alignment(both parent and children should use it)
+    // Elements alignment (both parent and children should use it)
     display: inline-block
 
     & > *
         display: inline-block
         font-size: inherit
-        padding: .6em
+        padding: .15em .6em .15em .6em
```

Comment:
If top/bottom and right/left are having the same values, you can specify only two first.
You don't need explicitly set the same values second times for bottom/rigtht